<launch>

<param name="/suturo_manipulation_tf_publisher/cam_frame" type="string" value="/laser_tilt_mount_link" />
<param name="/suturo_manipulation_tf_publisher/cam_x" type="double" value="-0.04" />
<param name="/suturo_manipulation_tf_publisher/cam_y" type="double" value="-0.26" />
<param name="/suturo_manipulation_tf_publisher/cam_z" type="double" value="0.0" />

  <node pkg="suturo_manipulation_tools" name="suturo_manipulation_tf_publisher" type="publish_objects_tf_frames" respawn="true" output="screen"/>

</launch>
